relation_engine_server/main.py

```diff
@@ -18,6 +18,26 @@
 app.register_blueprint(api_v1, url_prefix='/api/v1')
 
 
+def return_error(error_dict, code):
+    """return the appropriate error structure and code
+
+    Errors returned by the server have the basic format
+
+    'error': {
+        'message': <text explanation of error>,
+        'status': <HTTP error code>,
```

Comment:
It doesn't seem like we need status here if it is always in the header.

Reply:
Yeah, that was my thought too - the only reason I left it in is that it was in several of the error structures. I'm happy to remove it globally.